megalinter/descriptors/action.megalinter-descriptor.yml

```diff
@@ -0,0 +1,25 @@
+descriptor_id: ACTION
+descriptor_type: tooling_format
+descriptor_flavors:
+  - all_flavors # Applicable to CI in any language project
+file_extensions:
+  - ".yml"
+  - ".yaml"
+test_folder: action_actionlint
+linters:
+  # ACTIONLINT
+  - linter_name: actionlint
+    linter_url: https://rhysd.github.io/actionlint/
```

Comment:
Please add linter_text saying that actionlint lints github action workflows , it's better for documentation :)

Reply:
Solved with commit 5f25d0d393184f90d8739a647e0b5bd12142f8a2